src/modules/copy/copy.cpp

```diff
@@ -352,9 +355,6 @@ void DavixCopyInternal::copy(const Uri &src, const Uri &dst,
 
     delete request;
 }
-
```

Comment:
Line deletion again, but this one removes any empty lines between the two functions.